src/Cryptol/TypeCheck/Solver/Numeric/Sampling/Q.hs

```diff
@@ -0,0 +1,38 @@
+{-# LANGUAGE GeneralizedNewtypeDeriving #-}
+
+module Cryptol.TypeCheck.Solver.Numeric.Sampling.Q where
```

Comment:
Why define this separately from `Rational`?

Reply:
it started off looking a bit different, but actually yeah this is just `Rational`. I'll refactor it away